baseapp/baseapp_reactions/models.py

```diff
@@ -114,7 +115,6 @@ def update_reactions_count(self, target):
 
 class Reaction(AbstractBaseReaction):
     class Meta:
```

Comment:
can you inherit the meta class here `class Meta(AbstractBaseReaction.Meta)` and also delete the migration file and making migrations again?